queries.sql

```diff
@@ -40,3 +40,61 @@ SELECT *
 FROM animals
 WHERE weight>=10.4 and weight<=17.3 
 
+/**ex-2*/
+/*Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.*/
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon'
+
+/*Update the animals table by setting the species column to pokemon for all animals that don't have species already set*/
+UPDATE animals
+SET species="pokemon"
+WHERE species IS NULL;
```

Comment:
- [ ] You should put this Inside a transaction and  Commit the transaction, it may look like this :point_down: 

```sql
BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```